Avoid the CMake module provided by libxml2 which brings in transitive
dependencies.
--- CMakeLists.txt.orig	2024-11-09 17:36:34
+++ CMakeLists.txt	2024-11-09 17:43:09
@@ -411,7 +411,7 @@
 	# So, try first to find the CMake module provided by libxml2 package, then fallback
 	# on the CMake's FindLibXml2.cmake module (which can lack some definition, especially
 	# in static build case).
-	find_package(LibXml2 QUIET NO_MODULE)
+	find_package(LibXml2 QUIET)
 	if(DEFINED LIBXML2_VERSION_STRING)
 		set(LIBXML2_FOUND ON)
 		set(LIBXML2_INCLUDE_DIR ${LIBXML2_INCLUDE_DIRS})
